lib/impure/re.nim

```diff
@@ -25,6 +25,8 @@
 ## .. include:: ../doc/regexprs.txt
 ##
 
+{.deprecated.}
```

Comment:
Then I guess re should be undeprecated because it's deprecated already. Removed my duplicate deprecated pragma anyay.